relationships/synthesis/EBPF-CLIENT-to-EBPF_REDIS.yml

```diff
@@ -0,0 +1,82 @@
+relationships:
+  - name: clientConnectsToRedisServerFromServerSpan
+    version: "1"
+    origins:
+      - OpenTelemetry
```

Comment:
Just learned that, for `Span` events the origin should be `Distributed Tracing`.